build.yaml

```diff
@@ -0,0 +1,22 @@
+builders:
+  resourceBuilder:
```

Comment:
[nit] We don't have a documented convention, but most of our builders have been named with underscores since that's more typical in a yaml file.